src/astx/transpilers/python.py

```diff
@@ -113,3 +113,57 @@ def visit(self, node: astx.VariableAssignment) -> str:
         target = node.name
         value = self.visit(node.value)
         return f"{target} = {value}"
+
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: Type[astx.Float16]) -> str:
+        """Handle Float nodes."""
+        return "float"
+
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: Type[astx.Float32]) -> str:
+        """Handle Float nodes."""
+        return "float"
+
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: Type[astx.Float64]) -> str:
+        """Handle Float nodes."""
+        return "float"
+
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: astx.LiteralFloat16) -> str:
+        """Handle LiteralFloat nodes."""
+        return str(node.value)
+
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: astx.LiteralFloat32) -> str:
+        """Handle LiteralFloat nodes."""
+        return str(node.value)
+
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: astx.LiteralFloat64) -> str:
+        """Handle LiteralFloat nodes."""
+        return str(node.value)
+
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: Type[astx.Complex32]) -> str:
+        """Handle Complex32 nodes."""
+        return "(float, float)"
+
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: Type[astx.Complex64]) -> str:
+        """Handle Complex64 nodes."""
+        return "(float, float)"
```

Comment:
this will be used just for typing annotation

```suggestion
        return "Complex"
```